### PR TITLE
Ability to add a suffix to the docker tag image

### PR DESCRIPTION
**Pull request by mauriziovitale**

### CODE REVIEW

**Review by eromano — APPROVED** [review]
LGTM

File: .travis.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ stages:
   - name: test
     if: commit_message !~ /\[skip tests\]/
   - name: docker_latest
-    if: fork = false AND type != pull_request AND (branch = master OR branch =~ /release\/.*/)
+    if: fork = false AND type != pull_request AND (branch = master OR branch =~ /release\/.*/ OR branch = /feature\/.*/)
```
Comment (andrea-ligios) [inline .travis.yml:40]: `=~` is needed to run equality on regular expressions (in place of `=`)

File: tests/pipeline-all-amps/repo/pom.xml [hunk 2]
```diff
@@ -36,14 +37,14 @@
...
-<!--                  <artifactId>alfresco-governance-services-enterprise-repo</artifactId>-->
-<!--                  <version>${alfresco.ags.version}</version>-->
-<!--                  <type>amp</type>-->
-<!--                  <overWrite>false</overWrite>-->
-<!--                  <outputDirectory>${project.build.directory}/amps</outputDirectory>-->
-<!--                </artifactItem>-->
+               <!-- <artifactItem>
+                  <groupId>org.alfresco</groupId>
+                  <artifactId>alfresco-governanservices-enterprise-repo</artifactId>
+                  <version>${alfresco.ags.versi</version>
```
Comment (andrea-ligios) [inline tests/pipeline-all-amps/repo/pom.xml:43]: Here there's a tipo, `versi` should be `version}`

File: tests/pipeline-all-amps/repo/pom.xml [hunk 3]
```diff
@@ -36,14 +37,14 @@
...
-<!--                  <groupId>org.alfresco</groupId>-->
-<!--                  <artifactId>alfresco-governance-services-enterprise-repo</artifactId>-->
-<!--                  <version>${alfresco.ags.version}</version>-->
-<!--                  <type>amp</type>-->
-<!--                  <overWrite>false</overWrite>-->
-<!--                  <outputDirectory>${project.build.directory}/amps</outputDirectory>-->
-<!--                </artifactItem>-->
+               <!-- <artifactItem>
+                  <groupId>org.alfresco</groupId>
+                  <artifactId>alfresco-governanservices-enterprise-repo</artifactId>
```
Comment (andrea-ligios) [inline tests/pipeline-all-amps/repo/pom.xml:42]: Here there's a typo, `governanservices` should be `governance-services`

**Review by andrea-ligios — CHANGES_REQUESTED** [review]
Better not to leave typos even in commented parts 👍🏻

File: tests/pipeline-all-amps/repo/pom.xml [hunk 4]
```diff
@@ -36,14 +37,14 @@
                   <overWrite>false</overWrite>
                   <outputDirectory>${project.build.directory}/amps</outputDirectory>
                 </artifactItem>
-<!--                <artifactItem>-->
```
Comment (eromano) [inline tests/pipeline-all-amps/repo/pom.xml:39]: I think is better to create another folder with this thing always

**Review by eromano — CHANGES_REQUESTED** [review]

**Pull request closed without merging**